src/foam/u2/crunch/CapabilityCardView.js

```diff
@@ -76,12 +82,19 @@ foam.CLASS({
             'float': 'left'
           })
         .end()
-        .add(this.slot(function(cjStatus) {
-          return this.E().start(self.ReadOnlyEnumView, { data : cjStatus, clsInfo : cjStatus.cls_.LABEL.name, default : cjStatus.label })
-            .addClass(style.myClass('badge'))
-            .style({ 'background-color': cjStatus.background })
+        .add(this.slot(function(cjStatus, isRenewable) {
+          return this.E('span').start()
+            .style({ 'float' : 'right' })
+            .start(self.ReadOnlyEnumView, { data : cjStatus, clsInfo : cjStatus.cls_.LABEL.name, default : cjStatus.label })
+              .addClass(style.myClass('badge'))
+              .style({ 'background-color': cjStatus.background })
+            .end()
+            .start()
+              .addClass(style.myClass('renewable-description'))
+              .add(isRenewable ? "Capability is renewable" : "")
+            .end()
           .end();
-        }))
+        }))      
```

Comment:
Please remove the trailing spaces on this line.

Reply:
In future, please respond to comments to make it easier for your reviewer to track. Something like "done" is usually sufficient.